hr_timesheet_activity_begin_end/__manifest__.py

```diff
@@ -0,0 +1,21 @@
+# Copyright 2015 Camptocamp SA - Guewen Baconnier
+# Copyright 2017 Tecnativa, S.L. - Luis M. Ontalba
+# License AGPL-3 - See http://www.gnu.org/licenses/agpl-3.0.html
+
+{'name': 'Timesheet Activities - Begin/End Hours',
+ 'version': '12.0.1.0.0',
+ 'author':  'Camptocamp, '
+            'Tecnativa, '
+            'Odoo Community Association (OCA)',
+ 'license': 'AGPL-3',
+ 'category': 'Human Resources',
+ 'depends': ['hr_timesheet_sheet',
+             ],
+ 'website': 'https://github.com/OCA/timesheet',
+ 'data': ['views/hr_analytic_timesheet.xml',
+          'views/hr_timesheet_sheet.xml',
+          ],
+ 'test': [],
```

Comment:
if you don't use the tag, just delete it

Reply:
Done